KuringApp/KuringApp/Setting/SettingView.swift

```diff
@@ -8,10 +8,23 @@
 import SwiftUI
 import ComposableArchitecture
 
+enum InformationURL: String {
+    case kuringTeam = "https://bit.ly/3v2c5eg"
+    case privacy = "https://kuring.notion.site/65ba27f2367044e0be7061e885e7415c"
+    case policy = "https://kuring.notion.site/e88095d4d67d4c4c92983fd85cb693b9"
+    case instagram = "https://instagram.com/kuring.konkuk"
+}
+
 struct SettingListFeature: Reducer {
+    
     struct State: Equatable {
         // TODO: 나중에 디펜던시로
         var currentAppIcon: KuringIcon?
+        @BindingState var isCustomAlarmOn: Bool = false
+        
+        @PresentationState var webViewOpen: InformationWebViewFeature.State?
+        @PresentationState var feedbackViewPresent: FeedbackFeature.State?
```

Comment:
```suggestion
        @PresentationState var informationWebView: InformationWebViewFeature.State?
        @PresentationState var feddback: FeedbackFeature.State?
```

Feature 앞에 해당하는 네이밍으로 쓰는게 어떨까요?